tests/console/runc.pm

```diff
@@ -0,0 +1,103 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test runc installation and extended usage
+#    Cover the following aspects of runc:
+#      * package can be installed
+#      * create specification files
+#      * run the container
+#      * complete lifecycle (create, start, pause, resume, kill, delete)
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+use base "consoletest";
+use testapi;
+use utils;
+use strict;
+
+# Setup the required testing environment
+sub setup {
+    # install the docker package if it's not already installed
+    zypper_call('in docker');
+
+    # make sure docker daemon is running
+    systemctl('start docker');
+    systemctl('status docker');
+}
+
+sub create_oci_bundle {
+    # create the rootfs directory
+    assert_script_run('mkdir ./rootfs');
+
+    # export busybox via Docker into the rootfs directory
+    assert_script_run('docker export $(docker create busybox) | tar -C rootfs -xvf -');
+}
+
+sub run {
+    select_console("root-console");
+
+    # runC cannot create or extract the root filesystem on its own. Use Docker to create it.
+    record_info 'Setup', 'Setup the environment';
+    setup;
+    create_oci_bundle;
+
+    # installation of runc package
+    record_info 'Test #1', 'Test: Installation';
+    if (is_caasp && check_var('FLAVOR', 'DVD') && !check_var('SYSTEM_ROLE', 'plain')) {
```

Comment:
You should be able to use is_caasp('DVD') - you don't have to check flavor separately

Reply:
Ah, yes! That's even more *human-readable* ;)

So, if I understood correctly, I have to replace:
`if (is_caasp && check_var('FLAVOR', 'DVD') && !check_var('SYSTEM_ROLE', 'plain')) {`
with:
`if (is_caasp('DVD') && !check_var('SYSTEM_ROLE', 'plain')) { `

is this correct?

Reply:
Correct.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/220e44a63d410ef6ef35dda95ea649499fb0dbbf/lib/utils.pm#L273-L287